`zerocopy` 0.6.2 is not semver compatible with `0.6.1`: use zerocopy::U32;    |     ^^^^^^^^^^^^^ no `U32` in the root

Hi, thank you for this crate. 

# Problem 

When `zerocopy` `0.6.2` was released to crates.io our CI tests started failing like this:

```
error[E0432]: unresolved import `zerocopy::U32`
  --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/apache-avro-0.15.0/src/duration.rs:19:5
   |
19 | use zerocopy::U32;
   |     ^^^^^^^^^^^^^ no `U32` in the root
   |
help: consider importing one of these items instead
   |
19 | use digest::consts::U32;
   |     ~~~~~~~~~~~~~~~~~~~
19 | use rand::seq::index::IndexVec::U32;
   |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
19 | use rand::seq::index::IndexVecIntoIter::U32;
   |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
19 | use rand::seq::index::IndexVecIter::U32;
   |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     and 1 other candidate
```

We can fix our build by pinning to `zerocopy 0.6.1`

Here is the downstream ticket: https://github.com/apache/arrow-datafusion/issues/7221

Note that DataFusion uses `zerocopy` transitively the avro crate so I can't change the use of `zerocopy` directly

# Requested solution

If you agree that `0.6.2` is not semver compatible with 0.6.1, can you please [yank](https://doc.rust-lang.org/cargo/commands/cargo-yank.html) the `0.6.2` crate from crates.io

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@
 [package]
 edition = "2021"
 name = "zerocopy"
-version = "0.6.3-alpha"
+version = "0.6.3"
 authors = ["Joshua Liebow-Feeser <joshlf@google.com>"]
 description = "Utilities for zero-copy parsing and serialization"
 license = "BSD-2-Clause"
@@ -38,7 +38,7 @@ simd-nightly = ["simd"]
 __internal_use_only_features_that_work_on_stable = ["alloc", "simd"]
 
 [dependencies]
-zerocopy-derive = { version = "=0.6.3-alpha", path = "zerocopy-derive" }
+zerocopy-derive = { version = "=0.6.3", path = "zerocopy-derive" }
 
 [dependencies.byteorder]
 version = "1.3"

--- a/zerocopy-derive/Cargo.toml
+++ b/zerocopy-derive/Cargo.toml
@@ -5,7 +5,7 @@
 [package]
 edition = "2021"
 name = "zerocopy-derive"
-version = "0.6.3-alpha"
+version = "0.6.3"
 authors = ["Joshua Liebow-Feeser <joshlf@google.com>"]
 description = "Custom derive for traits from the zerocopy crate"
 license = "BSD-2-Clause"

